### DISCUSSION TITLE
Interactsh Options

### DISCUSSION THREAD

**Discussion by sec13b** [projectdiscovery/interactsh#813] [Q&A]
no offense what is the correct Server URL.
Thank you

![image](https://github.com/projectdiscovery/interactsh/assets/123419537/02733e8d-62dd-47f7-b4a4-8500e9674339)

**Comment by olearycrew** [comment]
The default server URLs are:

`oast.pro,oast.live,oast.site,oast.online,oast.fun,oast.me`

**Comment by olearycrew** [comment]
What application is this that you're using?

**Comment by sec13b** [comment]
i think oast.pro 
i just want to learn.
